content/_includes/ReplicatedEncryptedUnlock.md

```diff
@@ -18,18 +18,11 @@ Uncheck properties when replicating so that the destination dataset will not be
 4. Click *Save*.
 {{< /tab >}}
 {{< tab "Method 3: Replicate Key Encrypted Dataset/zvol" >}}
-Check **Full Filesystem Replication** so that the destination dataset will use the exported Encryption key from the source pool/dataset to unlock.
-1. Go to **Storage -> pool/root dataset**. Click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i>  and select **Export Key**.
-2. Download the key, open the text file, and copy the Key code.
-3. Go to **Data Protection** and click *ADD* in the *Replication Tasks* window.
-4. Click *Advanced Replication Creation*.
-5. Fill out the form as needed and make sure to enable *Full Filesystem Replication*.
-6. Click *Save*.
-7. On the receiving pool/dataset:
-  * Click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i>  next to pool/dataset and select *Unlock*.
-  * Unset Unlock with Key file.
-  * Paste the Key Code into Dataset Key. (if there is a space character at the end of the key - delete the space.)
-  * Click *Save*.
-  * Click *Continue*.
+1. Go to **Storage -> pool/root dataset** on the replication system. Click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i>  and select **Export Key**.
+2. Either download the Key file, open the file, and change the poolname/dataset to the recieving poolname/dataset or copy the Key code provided in the Key window.
```

Comment:
this one reads awkwardly. It sounds like the action is to apply a key file or code to a dataset. If that is the case...
```suggestion
2. Apply the key file or key code to the dataset. You can either download the key file, open that file and change the *pool name/dataset* to the receiving*pool name/dataset*, or copy the key code provided in the **Key** window.
```